website/docs/r/pki_secret_backend_config_auto_tidy.html.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: "vault"
+page_title: "Vault: vault_pki_secret_backend_config_auto_tidy resource"
+sidebar_current: "docs-vault-resource-pki-secret-backend-config-auto-tidy"
+description: |-
+  Sets the Auto Tidy configuration on a PKI Secret Backend for Vault.
+---
+
+# vault\_pki\_secret\_backend\_config\_auto\_tidy
+
+Allows setting the Auto Tidy configuration on a PKI Secret Backend
+
+## Example Usage
+
+```hcl
+resource "vault_mount" "pki" {
+  path                      = "pki"
+  type                      = "pki"
+  default_lease_ttl_seconds = 3600
+  max_lease_ttl_seconds     = 86400
+}
+
+resource "vault_pki_secret_backend_config_auto_tidy" "test" {
+  backend = vault_mount.pki.path
+  enabled = true
+  tidy_cert_store = true
+  interval_duration = "1h"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported (at least one tidy operation is required):
```

Comment:
I believe this is missing the common `namespace` and `backend` arguments

Reply:
Added.